### ISSUE TITLE
Dragging a tab away from main window, to create new the color formatting is gone

### ISSUE THREAD

**Issue by nivs1978** [notepad-plus-plus/notepad-plus-plus#3451]
#

### Description of the Issue
When I drag a tab away from the main window to create a new, the color formatting of the text disappears.
This only happen to file types added in the userDefineLang.xml file.

### Steps to Reproduce the Issue
1. Add a userDefinedLang.xml file to the %appdata%\Notepad++\ folder. Example use the one provided for MapBasic (*,mb files) here: http://www.twiav.nl/php/mapbasic.php
2. Open a few file, where one is an .mb file.
3. Drag one of the tabs away from the main window to create a new MapBasic window. All color formatting is gone.

### Expected Behavior
I expect to see the same color formatting as I do in the main window.

### Actual Behavior
The text of the .mb file is displayed in black and white.

### Debug Information

Notepad++ v7.4.1   (32-bit)
Build time : May 18 2017 - 02:00:37
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : mimeTools.dll NppConverter.dll NppExport.dll NppFTP.dll PluginManager.dll SpellChecker.dll XMLTools.dll 

![main_window](https://user-images.githubusercontent.com/1687820/27640053-6e1ab5aa-5c18-11e7-95e8-22961b9bc69e.png)
![new_window](https://user-images.githubusercontent.com/1687820/27640064-78006d4e-5c18-11e7-9bbb-15a8e98406fc.png)

Save this as test.mb to check color formatting:
'**************************************************************************
'                Include files and Define statements

Include "mapbasic.def" 'standard defines 

Include "auto_lib.def" 'include this file to access the AUTO_LIB library 
                       'The actual AUTO_LIB routines are in AUTO_LIB.MB[O]
Include "menu.def"
Include "icons.def"

Include "Library\Defaults.def"
Include "Library\ProgramInfo.def"
Include "Library\DEBUGLib.def"
Include "Library\ERRORLib.def"
Include "Library\MAPPERLib.def"
Include "Library\RIBBONLib.def"
Include "Library\TABLELib.def"

**Comment by BreakTheWind** [comment]
26 days and no resolution. I have the same problem. What gives? I just installed notepad++ and I have no added plugins.

Note that this happens only with user-defined languages.
Maybe a workaround would add the language to the default ones?

**Comment by ArkadiuszMichalski** [comment]
Still valid in latest Notepad++ 7.9. STR:
1. Download  [test.txt](https://github.com/notepad-plus-plus/notepad-plus-plus/files/5424025/test.txt) and change .txt extension to .md (NPP has in-build UDL for markdown).
2. Run Notepepad++ and open `test.md`.
3. Move/Open tab with `test.md` file in new instance, for example by drag this tab outside NPP main window or select proper commant from tab's context menu.

**Comment by bReathl3sS** [comment]
Same as @ArkadiuszMichalski - still not working in 7.9.1 64-bit Win10
Possible duplicate of #4769 #5589

**Comment by pryrt** [comment]
user in forum reported it: https://community.notepad-plus-plus.org/topic/20812/language-style-changes-when-tab-view-is-dragged-out-of-notepad

I confirmed with fresh v7.9.3 portable that the problem still exists:
1. create new file with some markdown in it
2. **Language > Markdown (preinstalled)**, 
3. save as `blah.md` 
4. verify the status bar says "user defined language file - markdown (preinstalled)" 
![image](https://user-images.githubusercontent.com/17455758/109514159-94619300-7a5a-11eb-95ec-719bb206df7e.png)
5. right click on the tab, **Move to New Instance**
6. now it shows 
![image](https://user-images.githubusercontent.com/17455758/109514257-ae02da80-7a5a-11eb-8816-5565570fe15d.png)

Debug Info: 
```
Notepad++ v7.9.3   (64-bit)
Build time : Feb  8 2021 - 18:16:35
Path : C:\usr\local\apps\npp\npp.7.9.3.portable.x64\notepad++.exe
Admin mode : OFF
Local Conf mode : ON
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1903
OS Build : 18362.1256
Current ANSI codepage : 1252
Plugins : mimeTools.dll NppConverter.dll NppExport.dll 
```

**Comment by pryrt** [comment]
Looking at #4883, I was surprised when the user there claimed the problem didn't exist in 7.9RC -- when I do a fresh unzip of the released v7.9 (I don't have a copy of the RC any more, and after release the RC URL at http://download.notepad-plus-plus.org/repository/7.x/7.9.RC/ doesn't work).  But when I try with the released v7.9 portable, I get the same bad results -> doing **Move to New Instance** changed the markdown file from `User Defined Language file - Markdown (preinstalled)` as it should be to `User Defined Language file` (ie, the default **Language > User-defined**)

```
Notepad++ v7.9   (64-bit)
Build time : Sep 22 2020 - 03:19:04
Path : C:\usr\local\apps\npp\npp.7.9.portable.x64\notepad++.exe
Admin mode : OFF
Local Conf mode : ON
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1903
OS Build : 18362.1256
Current ANSI codepage : 1252
Plugins : mimeTools.dll NppConverter.dll NppExport.dll 
```
